File: Javascript/02-longest-word.js
console.log(longestWord("abc def abcde"));

function longestWord(sentence) {
	var words = sentence.split(" ");
	var longestCount = 0;
	var longestWord = "";

	for (var i = 0; i < words.length; i++) {
		var wordCount = words[i].length;
		if (wordCount > longestCount) {
			longestCount = wordCount;
			longestWord = words[i];
		}
	}

	return longestWord;
}

File: Python/02-longest-word.py
def longest_word(sentence):
    words = sentence.split(" ")
    longest_idx = 0

    for i in range(1, len(words)):
        if len(words[i]) > len(words[longest_idx]):
            longest_idx = i

    return words[longest_idx]

print(longest_word("abc def abucee"))